src/main/java/net/camotoy/bedrockskinutility/client/data/CustomCapeData.java

```diff
@@ -0,0 +1,6 @@
+package net.camotoy.bedrockskinutility.client.data;
+
+import net.minecraft.resources.ResourceLocation;
+
+public record CustomCapeData(ResourceLocation cape) {
```

Comment:
For one parameter, having a whole class seems unnecessary.

Reply:
This class can be deleted.

Reply:
> This class can be deleted.

done